### PR TITLE
Add Southampton December Tech Day with provisional location

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

**Comment by RealOrangeOne** [comment]
Duplicate of #55?

**Comment by Adimote** [comment]
I have no access to push to #55, so I created a fresh request

**Comment by Adimote** [comment]
I suggest we push this now, email the relevant teams, then create a news post when we actually have the location confirmed, so we don't create a news post twice.

**Comment by Adimote** [comment]
@rspanton @jimmythompson @PeterJCLaw

File: _events/sr2017/southampton-tech-day-december.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Southampton December Tech Day 
+date: 2016-12-17 09:00:00
+layout: event
+type: tech day
```
Comment (jimmythompson) [inline _events/sr2017/southampton-tech-day-december.md]: I'll merge and then sort it, just something to remember in future. 👍

Comment (PeterJCLaw) [inline _events/sr2017/southampton-tech-day-december.md]: Is there something we can do to make it easier to get this sort of thing right? Perhaps a list of the proper types in the README or something the CI can do to validate a list? (Or both, though we'd want to ensure that there was only actually one canonical list and everything else referenced that).

**Pull request merged**